examples/experimental/onnx/semantic_segmentation/onnx_ptq_segmentation.py

```diff
@@ -76,8 +77,11 @@ def run(onnx_model_path: str, output_model_path: str, dataset_name: str,
     parser.add_argument("--data",
                         help="Path to dataset",
                         type=str)
-    parser.add_argument("--input_shape", help="Model's input shape",
-                        nargs="+", type=int, default=[1, 3, 768, 960])
+    parser.add_argument(
+        "--input_shape", help="Model's input shape. e.g. [1, 3, 768, 960]",
+        nargs="+", type=int, default=None)
+    parser.add_argument(
+        "--input_keys", help="Model's input key.", type=str, default=None)
```

Comment:
Furthermore, it is not evident in which format this "key", or multiple "keys" should be given in.

Reply:
Added an example similar to `input_shape`.